js/adapt-contrib-boxMenuGroupView.js

```diff
@@ -0,0 +1,25 @@
+define([
+  'core/js/views/menuItemView',
+  './adapt-contrib-boxMenuItemView'
+], function(MenuItemView, BoxMenuItemView) {
+
+  var BoxMenuGroupView = MenuItemView.extend({
+
+    postRender: function() {
+      _.defer(function() {
+        this.addChildren();
+      }.bind(this));
```

Comment:
```suggestion
      _.defer(this.addChildren.bind(this));
```
Would work OK here I think? Or:
```js
      _.defer(() => {
        this.addChildren();
      });
```
?

Reply:
(personally I think the first suggestion is clearer)

Reply:
I need to give Guy a run down on ES6, I realised early today that we haven't really done all that much on up-skilling with the ES6 stuff etc, so I'll give that a shot over the next couple of weeks or so.

Reply:
I think it would also be useful for us all to start considering a) when to use ES6 and b) how to use it...

Then 'when' is nicely illustrated by the two examples above i.e. the ES5 version is probably better in that instance than the ES6 one.

The 'how' would be along the lines of whether you should do this:
```js
_.defer(() => {
  this.addChildren();
});
```
or this:
```js
_.defer(() => this.addChildren());
```
i.e. a kind of ES6 styleguide... I have more like this too (and I'm sure I'm barely scratching the surface) but we can pick that up elsewhere!

Reply:
Amended to ES5. ES6 conversion can be done as a separate issue.

Reply:
@guywillis TBH I think the ES5 version is actually neater here. No need to go switching code to ES6 unless it actually makes it more readable/better.